File: js/src/formulas/infiniteSeries3.ts
export default function infiniteSeries3 (x: number) {
  if (x > 1 || x < -1 || x === 1 || x === -1) {
    console.log('Check |x| < 1')
  } else {
    return (1 / (1 - x))
  }
}

File: cs/src/formulas/infiniteSeries3.cs
public static class InfiniteSeries3
{
    public static double Calculate(double x)
    {
        if (x > 1 || x < -1 || x == 1 || x == -1)
            throw new ArgumentException("Check |x| < 1");
        return 1 / (1 - x);
    }
}
